utils/vlog/tests/CMakeLists.txt

```diff
@@ -0,0 +1,14 @@
+include(v2x_tests.cmake)
+
+add_subdirectory(dsp_combinational)
+add_subdirectory(dsp_inout_registered)
+add_subdirectory(dsp_inout_registered_dualclk)
```

Comment:
These tests below should be eventually uncommented.

Reply:
https://github.com/SymbiFlow/symbiflow-arch-defs/issues/599